### ISSUE TITLE
acbuild run: readlink /etc/localtime: invalid argument

### ISSUE THREAD

**Issue by asbachb** [appc/acbuild#132]
Hi,

I try to build the nginx example but script fails. Unsure if it's really an issue of acbuild or related to the alpine base image...

```
vm@ubuntu:~/acbuild/examples/nginx$ acbuild version
acbuild version v0.2.1-6-gbb2640c
appc version 0.7.1+git

vm@ubuntu:~/acbuild/examples/nginx$ sudo sh -x build-nginx.sh 
+ set -e
+ [  -ne 0 ]
build-nginx.sh: 4: [: Illegal number: 
+ acbuild --debug begin
Beginning build with an empty ACI
+ trap acbuildEnd EXIT
+ acbuild --debug set-name example.com/nginx
Setting name of ACI to example.com/nginx
+ acbuild --debug dep add quay.io/coreos/alpine-sh
Adding dependency "quay.io/coreos/alpine-sh"
+ acbuild --debug run apk update
Running: [apk update]
Downloading quay.io/coreos/alpine-sh: [========================] 2.65 MB/2.65 MB
run: readlink /etc/localtime: invalid argument
+ acbuildEnd
+ export EXIT=1
+ acbuild --debug end
Ending the build
+ exit 1
```

**Comment by namsral** [comment]
I can replicate this issue on Ubuntu but not on CoreOS; on CoreOS the nginx example builds fine.

**Comment by asbachb** [comment]
I think the problem is that /etc/localtime is not a symlink on Ubuntu:

```
vm@ubuntu:/tmp$ ls -l /etc/localtime 
-rw-r--r-- 1 root root 2335 Oct 23 17:15 /etc/localtime
vm@ubuntu:/tmp$ readlink /etc/localtime
vm@ubuntu:/tmp$ 
```

It would be nice to now what's the result of these commands on your CoreOS installation.

**Comment by namsral** [comment]
CoreOS stable (835.7.0)

    $ readlink /etc/localtime
    ../usr/share/zoneinfo/UTC

**Comment by asbachb** [comment]
The feature seems to be new:

https://github.com/appc/acbuild/commit/615e5cae57e0b0bcebf4498a4c561770f89bc16f

**Comment by jonboulle** [comment]
I'm perplexed as to why this has never bitten us in rkt, since it follows basically the same codepath..

**Comment by iaguis** [comment]
> I'm perplexed as to why this has never bitten us in rkt, since it follows basically the same codepath..

Because in rkt [we ignore errors](https://github.com/coreos/rkt/blob/66047da5806bbe35817199d8e38312892202724f/stage1/init/init.go#L100)

**Comment by asbachb** [comment]
I tried #134. It resolves the problem.

**Comment by blalor** [comment]
This may also be fixed by #147.

**Comment by dgonyeo** [comment]
Should be fixed by #147. Someone complain if that's not the case.
